packaging/storaged.spec.in

```diff
@@ -39,6 +39,7 @@ BuildRequires: intltool
 %if %{with_libblockdev_part} == 1
 BuildRequires: libblockdev-part-devel
 %endif # with_libblockdev_part
+BuildRequires: libstoragemgmt-devel >= 1.3.0
```

Comment:
I think this can be squashed into the previous patch adding the functionality that requires this change.